src/Core/Services/ApiService.cs

```diff
@@ -377,6 +377,34 @@ public async Task<string> GetActiveBearerTokenAsync()
             return accessToken;
         }
 
+        public async Task<object> PreValidateSso(string identifier)
+        {
+            var path = "/account/prevalidate?domainHint=" + WebUtility.UrlEncode(identifier);
+            using (var requestMessage = new HttpRequestMessage())
+            {
+                requestMessage.Version = new Version(1, 0);
+                requestMessage.Method = HttpMethod.Get;
+                requestMessage.RequestUri = new Uri(string.Concat(ApiBaseUrl, path));
```

Comment:
Should use the `Identity` service base Api, is that the same as `ApiBaseUrl` value?

Reply:
You are correct, should be `Identity` - fixed